internal/loader/configuration_setting_loader.go

```diff
@@ -693,7 +718,10 @@ func getFeatureFlagFilters(acpSpec acpv1.AzureAppConfigurationProviderSpec) []ac
 	if acpSpec.FeatureFlag != nil {
 		featureFlagFilters = deduplicateFilters(acpSpec.FeatureFlag.Selectors)
 		for i := 0; i < len(featureFlagFilters); i++ {
-			featureFlagFilters[i].KeyFilter = FeatureFlagKeyPrefix + featureFlagFilters[i].KeyFilter
+			if featureFlagFilters[i].KeyFilter != nil {
```

Comment:
If user set snapshot in feature flag section, what  is expected behavior? Ignoring all `featureFlag.selector.snapshot` ?

Reply:
No, we go ahead to use it, get the snapshot, and retrieve the feature flags settings from it.